Chinese/大话西游.md

```diff
@@ -37,3 +37,5 @@
 [周星驰](Zhou/Zhou.md) ?
 
 [小熊维尼](xinwang/xin.md)?
+
+[谢霆锋](TingfengXie/TingfengXie.md)?
```

Comment:
If you just created the file `TingfengXie.md`, it looks like you may have forgotten to `git add` it.